src/aspire/volume/__init__.py

```diff
@@ -259,6 +260,31 @@ def rotate(self):
     def denoise(self):
         raise NotImplementedError
 
+    def save(self, mrcs_filepath, overwrite=False):
+        """
+        Save volume to disk as mrc file
+        """
+        with mrcfile.new(mrcs_filepath, overwrite=overwrite) as mrc:
+            mrc.set_data(self._data.astype(np.float32))
+
+        if self.dtype != np.float32:
+            logger.info(f"Volume with dtype {self.dtype} saved with dtype float32")
+
+    @staticmethod
+    def load(mrcs_filepath, dtype=np.float32):
+        """
+        Load an mrc file as a Volume instance.
+
+        :param mrcs_filepath: Data filepath to load.
+
+        :param dtype: Optionally specifiy data type. Defaults to dtype=np.float32.
+
+        :return: Volume instance.
+        """
+        with mrcfile.open(mrcs_filepath) as mrc:
```

Comment:
It might be problematic to give this option since we immediately create a `Volume` instance. In particular, `Volume.load('new_file.mrc', mode = 'w+')` will try to make a volume from an empty file and throw the error:

`AssertionError: Volume data should be ndarray with shape NxLxLxL or LxLxL.
`
What's a little strange is that if I then try to load the file again with `Volume('new_file.mrc', mode='r+')` it produces an empty volume of shape (1,0,0,0).

Reply:
'r' doesn't work? that was what I meant.  I would not expect the writer modes to work...

Reply:
'r' is the default mode of `mrcfile.open()`, so in a sense we're already allowing 'r'. I'm not sure it would be functionally different if we also allowed 'r+'. But 'w+' will give the error above.

Reply:
I must have misread; I thought that the default was `r+`.  My bad! Sorry for the extra noise.

Reply:
No worries. I learned things.